components/workspacekit/pkg/seccomp/notify.go

```diff
@@ -324,7 +346,10 @@ func (h *InWorkspaceHandler) Mount(req *libseccomp.ScmpNotifReq) (val uint64, er
 				if wait > iwsBackoffMaxWait {
 					wait = iwsBackoffMaxWait
 				}
+			} else {
+				break
```

Comment:
I don't understand this `break`: Wouldn't it stop iteration after the first non-error loop? :thinking:

Reply:
Ah, we loop over `iwsBackoffSteps` - that makes sense. :slightly_smiling_face: